### PR TITLE
Remove dead code, refactor

### PR DESCRIPTION
**Pull request by brianstrauch**
Checklist
---------
1. [CRUCIAL] Is the change for features that are already live in prod?
   * yes: ok

What
----
* Cleaned up some unused files that I noticed while working on other PRs
* Cleaned up all unused fixtures in the integration test framework
* Small typo fix

### CODE REVIEW

File: internal/cmd/ksql/command_cluster_configureacls.go [hunk 1]
```diff
@@ -37,35 +36,33 @@ func (c *ksqlCommand) newConfigureAclsCommand() *cobra.Command {
...
-	cluster, err := c.V2Client.DescribeKsqlCluster(ksqlCluster, environmentId)
+	// Get the Kafka Cluster
+	kafkaCluster, err := c.Context.GetKafkaClusterForCommand()
 	if err != nil {
 		return err
 	}
 
-	if cluster.Spec.KafkaCluster.Id != kafkaCluster.Id {
-		output.ErrPrintf(errors.KsqlDBNotBackedByKafkaMsg, ksqlCluster, cluster.Spec.KafkaCluster.Id, kafkaCluster.Id, cluster.Spec.KafkaCluster.Id)
+	if ksqlCluster.Spec.KafkaCluster.Id != kafkaCluster.ID {
```
Comment (MuweiHe) [inline internal/cmd/ksql/command_cluster_configureacls.go:56]: is there getter for ksql spec?

**Review by MuweiHe — APPROVED** [review]
thx!

File: internal/pkg/cmd/prerunner.go [hunk 2]
```diff
@@ -70,9 +70,11 @@ func (r *PreRun) Anonymous(command *CLICommand, willAuthenticate bool) func(*cob
 			return err
 		}
 
-		if err := command.Config.InitDynamicConfig(cmd, r.Config); err != nil {
+		command.Config.Config = r.Config
```
Comment (MuweiHe) [inline internal/pkg/cmd/prerunner.go:73]: is it certain command.config is not nil

Comment (brianstrauch) [reply internal/pkg/cmd/prerunner.go:73]: Yep, in all cases this isn't `nil`.

**Pull request merged**